dot/config_integration_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package dot
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestExportConfig(t *testing.T) {
+	cfg, cfgFile := newTestConfigWithFile(t)
+	require.NotNil(t, cfg)
+
+	genFile := NewTestGenesisRawFile(t, cfg)
+
+	cfg.Init.Genesis = genFile
+
+	err := InitNode(cfg)
+	require.NoError(t, err)
+
+	file := exportConfig(cfg, cfgFile.Name())
+	require.NotNil(t, file)
+	os.Remove(file.Name())
```

Comment:
Do you perhaps need to close `file`? Or is it already closed?

Reply:
This is closed in `writeConfig` which is inside `exportConfig`

Reply:
Can we perhaps change `exportConfig` to just return a file path then? It feels strange to return a closed `*os.File` I would say.